src/box/lua/schema.lua

```diff
@@ -2661,6 +2708,88 @@ space_mt.run_triggers = function(space, yesno)
     end
     builtin.space_run_triggers(s, yesno)
 end
+space_mt.drop_constraint = function(space, name, field)
+    check_space_arg(space, 'drop_constraint')
+    check_space_exists(space)
+    check_param(name, 'name', 'string')
+    if field == nil then
+        if builtin.box_tuple_constraint_drop(space.id, name, #name) ~= 0 then
+            box.error()
+        end
+        return
+    end
+    local fieldno = get_fieldno(space, field)
+    if builtin.box_field_constraint_drop(space.id, name, #name,
+                                         fieldno) ~= 0 then
+        box.error()
+    end
+end
+space_mt.create_constraint = function(space, name, func, field)
+    check_space_arg(space, 'create_constraint')
+    check_space_exists(space)
+    check_param(name, 'name', 'string')
+    local constr;
+    if (func == nil) then
+        constr = name
+    else
+        constr = {[name] = func}
+    end
+    constr = normalize_constraint(constr, "")
```

Comment:
I see other usages of this function employ `''` (single quotes). I suggest to do the same here and in `normalize_foreign_key()`.

Reply:
Fixed.